### DISCUSSION TITLE
Cannot join Discord channels

### DISCUSSION THREAD

**Discussion by DimMan88** [project64/project64#2399] [General]
### Expected behavior?

-

### Actual behavior?

As rules says, I OKed the rules and message the bot but my message is not sent so I cannot view the channels. 

### Steps to reproduce the problem

cannot join discord

### Specifications and build details

-

### Using included plugins

- [X] I can confirm I am using the included RSP, audio, input, and graphics plugins

### Plugins used while the issue occurred

-

### Copyright Compliance

- [X] There are no infringing files attached to this bug report.

**Comment by flagrama** [comment]
This is not relevant to the repository. Don't open bug reports for stuff unrelated to the application. You can use Discussions for that.

**Comment by DimMan88** [comment]
There is no other way to communicate for this.

**Comment by flagrama** [comment]
I literally just told you to use https://github.com/project64/project64/discussions

**Comment by flagrama** [comment]
Now that zilmar has moved this to the proper place, ensure you're messaging zilbot, that you have DMs open to everyone, and that you are typing the phrase exactly how it is written in the rules.

**Comment by project64** [comment]
what are you messaging to zilbot and what is discord user name

**Comment by DimMan88** [comment]
I will not request roms

**Comment by DimMan88** [comment]
Discordname is DimMan88

**Comment by DimMan88** [comment]
I message zilbot but discord tells me that the message cannot be sent because the recipient accepts DM only by friend etc.....

**Comment by DimMan88** [comment]
Solved. I had disabled direct messages.
